libr/bin/p/bin_dyldcache.c

```diff
@@ -841,8 +841,8 @@ static void carve_deps_at_address(RBuffer *cache_buf, cache_img_t *img, cache_hd
 				cmd == LC_LOAD_UPWARD_DYLIB) {
 			bool found;
 			const char *key = sdb_fmt ("%s", cursor + 24);
-			int dep_index = ht_pp_find (path_to_idx, key, &found);
-			if (!found) {
+			ut64 dep_index = (ut64) ht_pp_find (path_to_idx, key, &found);
```

Comment:
This wont work on 32 bit machines. You need to box the ut64. Or use ht_pu instead of ht_pp

Reply:
ok, so i just need pointer keys and int values (of which i care only about the lower 16 bits, actually), i think just using size_t is fine for this.